test/unit/worker.js

```diff
@@ -208,6 +210,7 @@ describe('Worker', function () {
               assert.notEqual(initDelay, worker.retryDelay, 'delay increased')
               assert.equal(worker.retryDelay, 4, 'delay increased to max')
               assert.equal(taskHandler.callCount, 4, 'task was called twice')
+              sinon.assert.calledWith(opts.task, {message: 'hello world'}, {attempt: 2, timeout: 0, retryDelay: 2})
```

Comment:
still pondering if this is an okay thing to do, but while I ponder, add whitespace to the insides of your curly brackets (consistency w/ the file; above as well)